cdap-proto/src/main/java/co/cask/cdap/proto/Id.java

```diff
@@ -27,7 +27,7 @@
  */
 public final class Id  {
 
-  private static boolean isId(String name) {
+  public static boolean isId(String name) {
```

Comment:
I think its fine to leave it public. It'll allow validation of streamName, without having to construct Id using namespace.